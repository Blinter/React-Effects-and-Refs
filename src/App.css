/* App.css */
body {
    font-family: 'Arial', sans-serif;
    margin: 0;
    padding: 0;
    background-color: #f0f0f0;
    color: #333;
  }
  
  #root {
    max-width: 800px;
    margin: 0 auto;
    padding: 20px;
  }
  
  h1 {
    text-align: center;
    color: #007bff;
    margin-bottom: 20px;
  }
  
  .loading {
    text-align: center;
    font-size: 1.2em;
    margin-top: 20px;
  }