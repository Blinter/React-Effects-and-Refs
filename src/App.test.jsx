// App.test.jsx

import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

describe('App component', () => {
    test('renders Deck component', () => {
        render(<App />);
    });
    test('matches snapshot', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
    });
});