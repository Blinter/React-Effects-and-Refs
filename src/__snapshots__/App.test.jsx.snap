// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches snapshot 1`] = `
<div>
  <h1>
    Card Flipper
  </h1>
  <div
    class="loading"
  >
    Loading...
  </div>
</div>
`;
